client/components/auth-form.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
You only need Bootstrap CSS import in one place and it will work across the entire app. So move this to the `App` component and delete it everywhere else.